ci_scripts/check_fauxpas.sh

```diff
@@ -20,3 +20,4 @@ fi
 echo "Linting with Faux Pas..."
 
 fauxpas check Stripe.xcodeproj/ --target "StripeiOSStatic" --configFile "./FauxPasConfig/main.fauxpas.json" --minErrorStatusSeverity Concern
```

Comment:
Oh I also remembered! You need to add `set -e` here so that the script exits if the first check fails.